src/simulation_lifecycle/utils/feature.c

```diff
@@ -0,0 +1,145 @@
+
+#include "cJSON.h"
+#include "stdlib.h"
+#include "string.h"
+#include "simulation_lifecycle/structures.h"
+
+cJSON * geojson_get_features(cJSON * geo_json) {
+    return  cJSON_GetObjectItem(geo_json, "features");
+}
+
+cJSON * feature_get_properties(cJSON * feature_json) {
+    return cJSON_GetObjectItem(feature_json, "properties");
+}
+
+cJSON * feature_get_property(cJSON * feature_json, char * property) {
+    cJSON * properties = feature_get_properties(feature_json);
+
+    return (properties == NULL) ? NULL : cJSON_GetObjectItem(properties, property);
+}
+
+char * feature_get_string_property(cJSON * feature_json, char * property) {
+    cJSON * p = feature_get_property(feature_json, property);
+
+    return (p == NULL) ? NULL : cJSON_GetStringValue(p);
+}
+
+int * feature_get_int_property(cJSON * feature_json, char * property) {
+    cJSON * p = feature_get_property(feature_json, property);
+
+    if (p == NULL) {
+        return NULL;
+    }
+
+    return &p->valueint;
```

Comment:
I would return `p->valueint`, so the function returns a copy of the integer instead of a pointer to the original integer.